sidebar/src/components/deploy/contract/params/DeployContractParams.logic.ts

```diff
@@ -29,7 +29,7 @@ export const useDeployContractsParams = (contracts: DeployContracts) => {
         form.resetField(`inputs.${idx}`);
       });
     };
-  }, [inputs]);
+  }, [inputs, form]);
```

Comment:
Avoid using form in dependencies, otherwise the inputs will be set every time form change and we don't want that